js/app.js

```diff
@@ -0,0 +1,39 @@
+const totalZombiesElement = document.querySelectorAll('.total-zombies')[0];
```

Comment:
use
const totalZombiesElement = document.querySelector('.total-zombies');
instead of
const totalZombiesElement = document.querySelectorAll('.total-zombies')[0];